types/corporatePurchase/corporatePurchase.d.ts

```diff
@@ -0,0 +1,306 @@
+    * @param merchantDisplayUrl [string]: public merchant icon (png image). ex: "https://sandbox.api.starkbank.com/v2/corporate-icon/merchant/ifood.png"
+    * @param merchantFee [integer]: fee charged by the merchant to cover specific costs, such as ATM withdrawal logistics, etc. ex: 200 (= R$ 2.00)
+    * @param methodCode [string]: method code. Options: "chip", "token", "server", "manual", "magstripe" or "contactless"
+    * @param tags [list of strings]: list of strings for tagging returned by the sub-issuer during the authorization. ex: ["travel", "food"]
+    * @param corporateTransactionIds [list of strings]: ledger transaction ids linked to this Purchase
+    * @param status [string]: current CorporateCard status. Options: "approved", "canceled", "denied", "confirmed", "voided"
+    * @param updated [string] latest update datetime for the CorporateCard. ex: '2020-03-10 10:30:00.000'
+    * @param created [string] creation datetime for the CorporateCard. ex: '2020-03-10 10:30:00.000'
+    */ 
+
+    readonly id : string
+    readonly holderId : string
+    readonly holderName : string
+    readonly centerId : string
+    readonly cardId : string
+    readonly cardEnding : string
+    readonly description : string
+    readonly amount : number
+    readonly tax : number
+    readonly issuerAmount : number
+    readonly issuerCurrencyCode : number
+    readonly issuerCurrencySymbol : number
+    readonly merchantAmount : number
+    readonly merchantCurrencyCode : string
+    readonly merchantCurrencySymbol : string
+    readonly merchantCategoryCode : string
+    readonly merchantCategoryType : string
+    readonly merchantCountryCode : string
+    readonly merchantName : string
+    readonly merchantDisplayName : string
+    readonly merchantDisplayUrl : string
+    readonly merchantFee : number
+    readonly methodCode : string
+    readonly tags : string[]
+    readonly corporateTransactionIds : string[]
+    readonly status : string
+    readonly updated : string
+    readonly created  : string
+
+    constructor(params?: {id?: string | null,  holderId?: string | null,  holderName?: string | null,  centerId?: string | null,  cardId?: string | null,  cardEnding?: string | null,  description?: string | null,  amount?: number | null,  tax?: number | null,  issuerAmount?: number | null,  issuerCurrencyCode?: number | null,  issuerCurrencySymbol?: number | null,  merchantAmount?: number | null,  merchantCurrencyCode?: string | null,  merchantCurrencySymbol?: string | null,  merchantCategoryCode?: string | null,  merchantCategoryType?: string | null,  merchantCountryCode?: string | null,  merchantName?: string | null,  merchantDisplayName?: string | null,  merchantDisplayUrl?: string | null,  merchantFee?: number | null,  methodCode?: string | null,  tags?: string[] | null,  corporateTransactionIds?: string[] | null,  status?: string | null,  updated?: string | null,  created})
```

Comment:
caraio pupilo olha o tamanho dessa linha KKKKKKKKKKKK
pensa q vc tem q bater o olho no codigo e conseguir ler tudo, se tem um linha desse tamanho n tem como ler
tenta deixar tudo em 80 
https://dev.to/brad_beggs/vs-code-vertical-rulers-for-prettier-code-3gp3